diagnosis/getkuberneteslogs.sh

```diff
@@ -78,6 +223,22 @@ do
             USER="$2"
             shift 2
         ;;
+        --spn-client-id)
+            SPN_CLIENT_ID="$2"
+            shift 2
+        ;;
+        --spn-client-secret)
+            SPN_CLIENT_SECRET="$2"
```

Comment:
Azure SDKs rely on a few environment variables for auth purposes
https://github.com/Azure/azure-sdk-for-go#more-authentication-details

Maybe we can follow that model? Sorry I just remembered

Reply:
Will look into it and make changes in next iteration